src/app/auth/login/login.component.html

```diff
@@ -1,2 +1,8 @@
-<p>login works!</p>
-<button (click)="handleClick()">Go next page</button>
+<h2>Runners</h2>
+<form action="#">
+  <input type="text" class="auth-form-field" placeholder="Логин">
```

Comment:
Please, never use magic number or string in future.It is horrible. You can create in component variable(for example: loginPlaceholder = Placeholder.LOGIN). In future for any strings you have to create ENUMS.